source/_posts/2024-06-05-release-20246.markdown

```diff
@@ -0,0 +1,731 @@
+[#118108]: https://github.com/home-assistant/core/pull/118108
+
+{% enddetails %}
+
+{% details "File" %}
+
+The notify services for the file integration now require that the file path
+is an allowed path. You should check the accessed file is in the
+[allowlist_external_dirs](/integrations/homeassistant/#allowlist_external_dirs)
+to ensure your automations keep working.
+
+([@jbouwh] - [#116861]) ([documentation](/integrations/file))
+
+[@jbouwh]: https://github.com/jbouwh
+[#116861]: https://github.com/home-assistant/core/pull/116861
+
+{% enddetails %}
+
+{% details "MQTT" %}
+
+MQTT Lock now supports the lock `open` and `opening` states. When a lock is used
+in `optimistic` mode, its state will become `open` instead of `unlocked` when
+it is opened.
+
+([@jbouwh] - [#117110]) ([documentation](/integrations/mqtt))
+
+[@jbouwh]: https://github.com/jbouwh
+[#117110]: https://github.com/home-assistant/core/pull/117110
+
+---
+
+The behavior on processing  empty and `"None"` state payloads for MQTT entities
+has been made more predictable for `alarm_control_panel`, `climate`, `cover`,
+`device_tracker`, `lock`, `select`, `valve`, and `water_heater` items:
+
+- A "None" (string) for a state payload generally means the state
+  becomes `unknown`.
+- An empty payload is ignored.
+
+Previously, a "None" state could be considered invalid or ignored.
+
+([@jbouwh] - [#117813]) ([documentation](/integrations/mqtt))
+
+[@jbouwh]: https://github.com/jbouwh
+[#117813]: https://github.com/home-assistant/core/pull/117813
+
+{% enddetails %}
+
+{% details "OpenWeatherMap" %}
+
+Migration to the new library with support of OneCall API (3.0).
+
+OWM API v2.5 will be closed in June 2024. All OpenWeatherMap integrations must
+be migrated to mode (API) v3.0 to avoid service interruption.
+
+Before the migration, you must have an active subscription (be aware that
+subscription activation takes up to 2h). After the subscription is activated,
+users can migrate integrations to mode v3.0 via repair notification or manually
+via the integration configuration menu.
+
+The subscription has a free tier with 1000 calls/day. Consider setting the limit
+on the OpenWeatherMap website to stay under the threshold where API usage
+incurs a cost.
+
+[OpenWeatherMap official statement](https://openweathermap.org/one-call-transfer)
+
+([@freekode] - [#116870]) ([documentation](/integrations/openweathermap))
+
+[@freekode]: https://github.com/freekode
+[#116870]: https://github.com/home-assistant/core/pull/116870
+
+{% enddetails %}
+
+{% details "Synology DSM" %}
+
+The option to define a custom timeout has been removed. The timeout is set
+static to a sane value of 30 seconds.
+
+([@mib1185] - [#116815]) ([documentation](/integrations/synology_dsm))
+
+[@mib1185]: https://github.com/mib1185
+[#116815]: https://github.com/home-assistant/core/pull/116815
+
+{% enddetails %}
+
+{% details "Webmin" %}
+
+You may need to remove obsolete duplicated entities if this change affects you.
```

Comment:
What the change is isn't mentioned.

Reply:
Seems to be intentional: https://github.com/home-assistant/home-assistant.io/pull/33098#discussion_r1628177210

Reply:
The sentence doesn't make sense in the current form. We don't need to explain the details to say what "the change" is.

Reply:
Yes, one of those is ok.

Reply:
https://github.com/home-assistant/home-assistant.io/pull/33164